plugins/obs-qsv11/obs-qsv11-plugin-main.c

```diff
@@ -60,11 +60,19 @@ OBS_DECLARE_MODULE()
 OBS_MODULE_USE_DEFAULT_LOCALE("obs-qsv11", "en-US")
 MODULE_EXPORT const char *obs_module_description(void)
 {
+#ifdef ENABLE_HEVC
+	return "Intel Quick Sync Video H.264/HEVC encoder support for Windows";
```

Comment:
"Intel Quick Sync Video encoder support for Windows" and remove #ifdef.

Reply:
Done.